src/implementations/document-store/transfer-ownership.ts

```diff
@@ -0,0 +1,39 @@
+import { DocumentStoreFactory } from "@govtechsg/document-store";
+import signale from "signale";
+import { getLogger } from "../../logger";
+import { DocumentStoreTransferOwnershipCommand } from "../../commands/document-store/document-store-command.type";
+import { getWalletOrSigner } from "../utils/wallet";
+import { dryRunMode } from "../utils/dryRun";
+import { TransactionReceipt } from "@ethersproject/providers";
+
+const { trace } = getLogger("document-store:transfer-ownership");
+
+export const transferOwnershipToDocumentStore = async ({
```

Comment:
hi, the naming seems to be different from the functionality and description. it is more towards transferDocumentStoreOwnershipToWallet ?

Reply:
hello! agreed and changed